.changeset/dull-walls-lie.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Empty addresses still display the "Hidden assets" badge, even though those assets are not part of the address tokens.

<img width="350" alt="image" src="https://github.com/user-attachments/assets/cbd534b1-6d53-4d24-80df-60487a80c666" />

Reply:
Fixed in my latest PR in commit 106094638